text_extensions_for_pandas/io/conll.py

```diff
@@ -700,9 +712,9 @@ def conll_2003_to_dataframes(input_file: str,
     * `ent_type`: Entity type names for tokens tagged "I" or "B" in
       the `ent_iob` column; `None` everywhere else.
     """
-    parsed_docs = _parse_conll_file(input_file, column_names, iob_columns)
-    doc_dfs = [_doc_to_df(d, column_names, iob_columns, space_before_punct)
-               for d in parsed_docs]
+    parsed_docs, doc_line_nums = _parse_conll_file(input_file, column_names, iob_columns)
```

Comment:
This changes on these 3 lines can go away if line numbers are included in the `_SentenceData` data structure.